src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Extensions/UserRequestModelExtensionsTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using EPR.SubsidiaryBulkUpload.Application.Extensions;
+
+namespace EPR.SubsidiaryBulkUpload.Application.UnitTests.Extensions;
+
+[TestClass]
+public class UserRequestModelExtensionsTests
```

Comment:
Will probably need to update these tests to use Guids

Reply:
Updated.